### PR TITLE
Feature/fix hide/show header

### PR DESCRIPTION
**Pull request by TatianaLopaeva**
Added new Layout with hide header on scroll down & show on scroll up on Tablet/Mobile

### CODE REVIEW

File: products/ASC.Files/Client/src/components/pages/Home/index.js [hunk 1]
```diff
@@ -129,7 +133,7 @@ class PureHome extends React.Component {
         if (filter) {
           const folderId = filter.folder;
           console.log("filter", filter);
-          return fetchFiles(folderId, filter);
+          return fetchFiles(folderId, filter, this.isResize);
```
Comment (AlexeySafronov) [inline products/ASC.Files/Client/src/components/pages/Home/index.js:136]: Replce useless param isResize. Think to redux way.

File: web/ASC.Web.Common/src/components/NavMenu/index.js [hunk 2]
```diff
@@ -16,10 +16,17 @@ import { getLanguage } from "../../store/auth/selectors";
...
+const { size } = utils.device;
 const StyledContainer = styled.header`
   align-items: center;
   background-color: ${backgroundColor};
+
+  ${(props) =>
+    props.isTablet &&
+    !props.isLoaded &&
+    css`
+      margin-right: -16px;
```
Comment (AlexeySafronov) [inline web/ASC.Web.Common/src/components/NavMenu/index.js:28]: Add a comment for this

File: web/ASC.Web.Common/src/components/PrivateRoute/PrivateRoute.js [hunk 3]
```diff
@@ -19,6 +21,13 @@ const PrivateRoute = ({ component: Component, ...rest }) => {
     computedMatch,
   } = rest;
   const { userId } = computedMatch.params;
+  const isTablet = window.innerWidth <= size.tablet;
+
+  useEffect(() => {
+    isTablet &&
+      document.getElementById("scroll") &&
```
Comment (AlexeySafronov) [inline web/ASC.Web.Common/src/components/PrivateRoute/PrivateRoute.js:28]: Add a comment for this and move getElementId result into variable

Comment (AlexeySafronov) [reply web/ASC.Web.Common/src/components/PrivateRoute/PrivateRoute.js:28]: Rename id scroll to something different (more unique)

File: web/ASC.Web.Components/src/components/scrollbar/custom-scrollbars-virtual-list.js [hunk 4]
```diff
@@ -30,6 +30,7 @@ class CustomScrollbars extends React.Component {
         onScroll={onScroll}
         stype={stype}
         className={className}
+        id="scroll"
```
Comment (AlexeySafronov) [inline web/ASC.Web.Components/src/components/scrollbar/custom-scrollbars-virtual-list.js:33]: Already used id "scroll" inside scrollbar

File: web/ASC.Web.Components/src/components/scrollbar/index.js [hunk 5]
```diff
@@ -97,6 +97,7 @@ const Scrollbar = React.forwardRef((props, ref) => {
       style={{ ...style, ...view }}
       tabIndex={-1}
       className={"scroll-body"}
+      id="scroll"
```
Comment (AlexeySafronov) [inline web/ASC.Web.Components/src/components/scrollbar/index.js:100]: Already used id "scroll" inside custom-scrollbars-virtual-list

File: web/ASC.Web.Common/src/components/NavMenu/index.js (hunk 2 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Common/src/components/NavMenu/index.js]: It was added

**Pull request closed without merging**